docs/Project-Configuration/Project-Commands.md

```diff
@@ -403,11 +403,29 @@ Any updates to the expansions made with this command will only persist for the d
     updates:
     - key: artifact_url
       value: http://s3/static-artifacts.tgz
+
+- command: expansions.update
+  params:
+    updates:
+    - key: artifact_url
```

Comment:
This one doesn't showcase a new feature but using the `concat` param that we already had.